Dexterous-Rambutan/battle-code
# Psql stream import

#### What's this PR do?

Enables /api/resetChallenges route
The way the server resets the database by importing a csv file into Postgres directly via fscreateReadStream and pg-copy-streams node module. This method is much faster than hardcoding challenges via the Bookshelf Challenge object.
Also yak-shaved for deployment vs dev environment
#### What are the important parts of the code?
#### How should this be tested by the reviewer?

Since it's already deployed:
1. Go to http://104.131.141.22/
1. Log in 
1. Go to http://104.131.141.22/api/resetDBWithData
1. Got to http://104.131.141.22/api/resetChallenges
1. Go to http://104.131.141.22/ and nav to practice mode
1. Clicking on any of the challenge links should take you to a challenge
#### Is any other information necessary to understand this?
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

Closes #95 
#### Screenshots (if appropriate)
